### PR TITLE
Created profile page with all the UI and styling (with hard coded mock data/elements)

### PR DESCRIPTION
**Pull request by Julianna-Venter**

### CODE REVIEW

File: src/profilePage/ProfilePage.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  weekLabelAttributes,
+} from "./calendarHeatmap";
+
+function ProfilePage() {
+  return (
+    <div className="h-screen w-full flex flex-col justify-center items-center p-5 gap-5">
+      <BackgroundAssets />
+      <nav
+        id="Navbar"
+        className=" bg-off_white w-full h-[60px] rounded-2xl shadow-3xl flex items-center p-2.5 justify-between"
```
Comment (AshirMohab) [inline src/profilePage/ProfilePage.tsx]: convert the `px` values to `rem`

File: src/profilePage/ProfilePage.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              d="M17.593 3.322c1.1.128 1.907 1.077 1.907 2.185V21L12 17.25 4.5 21V5.507c0-1.108.806-2.057 1.907-2.185a48.507 48.507 0 0 1 11.186 0Z"
+            />
+          </svg>
+        </div>
+        {/* this is just for future use  */}
+        {/* solid bookmark */}
+        {/*<div
```
Comment (AshirMohab) [inline src/profilePage/ProfilePage.tsx:91]: This is fine, letting the reviewers know it is future work

**Review by AshirMohab — CHANGES_REQUESTED** [review]
Requesting changes for two things.
1. Convert your `px` values to `rems` since screen sizes and pixel density change. 
2. You can create a `mockData` file which will house your mock data. This all you to have the `dataStructures` already built and make it easier for you when you implement the props and api calls. (Optional but beneficial)

**Review by yanevdw — APPROVED** [review]
Looks good to me! Nicely done!

**Review by AshirMohab — APPROVED** [review]

**Review by AyaazStreet — COMMENTED** [review]
The code looks good to merge 🥳

**Pull request merged**